modules/ROOT/pages/performance-and-impact.adoc

```diff
@@ -1,27 +1,67 @@
 = Anypoint Monitoring Performance, Impact, and Limits
 
-Review details about the performance impact and data retention with Anypoint Monitoring.
+Anypoint Monitoring processes and retains massive amounts of data with a low performance impact. Titanium subscribers receive an enhanced data retention package and more storage than Gold and Platinum subscribers. Anypoint Monitoring users can also improve their user experience with adjustments to screen resolution and efficiently parsed lines of log data.
 
 == Mule Monitoring Agent Performance Impact
 
 Based on typical usage, the expected CPU utilization is less than 5%. For more information, refer to xref:monitoring::configure-monitoring-cloudhub#performance-impact[Configure and Enable Anypoint Monitoring for Mule Apps (CloudHub)].
 
-== Data Retention and Aggregation
+Anypoint Monitoring asynchronously collects data at a minute resolution, which does not block Mule runtime data processing.
 
-Anypoint Monitoring asynchronously collects data, which does not block Mule runtime data processing.
+== Data Retention and Limits
 
-* Minute Resolution for Metrics.
-* Gold and Platinum subscriptions:
- ** API analytics metrics data are retained for 90 days.
- ** Application metrics data are retained for 30 days.
+=== Gold and Platinum Subscription Limits
 
+Gold and Platinum subscribers to Anypoint Platform receive modest data retention and feature limits:
+
+Logs::
+Log limits are documented in xref:runtime-manager::viewing-log-data.adoc#log-persistence[Runtime Manager].
+
+Application Metrics::
+30 days.
+
+Enhanced API Metrics::
+30 days. The limit for data in the *Client Applications* tab is 1 hour.
+
+Basic Alerts::
+50 basic alerts.
+
+=== Titanium Subscription Limits
+
+Titanium subscribers to Anypoint Platform receive more robust data retention and feature limits:
+
+Logs::
+In raw format, your logs can occupy as much space as you receive with your Titanium subscription. Note that the storage space you receive with your Titanium subscription is used to store logs, metrics data, and search indexes. As indexes and metrics do not take up much storage capacity, the majority of the allocated storage can be used to store the raw log data.
```

Comment:
@michael-lee-mulesoft I don't think storage space purchased is consumed by the search index.  It is only consumed by the archived logs and metrics.  Search index occupies an additional storage above and beyond the purchased storage limit.

Reply:
@jiang-wu You are right.

@sarathecoubian `and search indexes` is not right. 
`As metrics do not take up much storage capacity`